app/controllers/api/v1/gardens_controller.rb

```diff
@@ -1,14 +1,34 @@
 class Api::V1::GardensController < ApplicationController
   before_action :set_garden, only: [:show]
+  before_action :authenticate_user, only: [:create]
```

Comment:
👍

Reply:
I figure this method will be useful for us over the next few days.